site2/tools/build-site.sh

```diff
@@ -58,7 +58,7 @@ rm -rf "$CONTENT_DIR"
 mkdir -p "$CONTENT_DIR"
 cp -R "$GEN_SITE_DIR"/reference "$CONTENT_DIR"
 # cp -R "$GEN_SITE_DIR"/api "$CONTENT_DIR"
-cp -R ./build/* "$CONTENT_DIR"
+cp -R ./build/ "$CONTENT_DIR"
 cp -R "$WEBSITE_DIR"/static/swagger/* "$CONTENT_DIR"/swagger/
```

Comment:
Perhaps we don't need this line since the static swagger is copyed at the line above.

Reply:
And we can add a `ls` command to debug why https://github.com/apache/pulsar-site/actions/runs/3317429325/jobs/5480293467 failed.

Reply:
fixed with https://github.com/apache/pulsar-site/pull/264